django_multitenant/middlewares.py

```diff
@@ -0,0 +1,14 @@
+from django_multitenant.utils import set_current_tenant
+
+from django_multitenant.views import get_tenant
+
+
+class MultitenantMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        if request.user and not request.user.is_anonymous:
```

Comment:
I see your point, but I am hesitant to rely solely on it. There may be certain cases that are not documented, and I want to ensure that we avoid any potential errors since we are dealing with multiple versions of Django. Furthermore, upon examining example codes for request handling, I have noticed that the code blocks include a reference to request.user. However, I do not believe it is our responsibility to address cases where the user does not exist. In fact, in some cases, it may be a valid scenario. Therefore, it is my opinion that the correct approach is to check for non-existence and skip the non-existence case.